lib/distillery/plugins/link_config.ex

```diff
@@ -17,13 +17,17 @@ defmodule Releases.Plugin.LinkConfig do
   use Mix.Releases.Plugin
 
 
-  def before_assembly(_), do: nil
+  def before_assembly(_, _), do: nil
```

Comment:
Should we bump required distillery version?

Reply:
These callback functions were added at distillery version 1.1.0.
(ref: https://github.com/bitwalker/distillery/commit/10db10947c982e4fe70ef6ba810db8bb085b968b)

I think it is not necessary to change required distillery version.